src/load/azext_load/data_plane/load_test/custom.py

```diff
@@ -0,0 +1,287 @@
+    )
+
+    if test_plan is not None:
+        upload_test_plan(client, test_id, test_plan, no_wait)
+
+    if configuration_files is not None:
+        upload_configuration_files(client, test_id, configuration_files)
+                    
+    return response_obj
+
+
+def update_test(
+    cmd,
+    test_id,
+    load_test_resource,
+    display_name=None,
+    resource_group_name=None,
+    load_test_config_file=None,
+    test_description=None,
+    test_plan=None,
+    configuration_files=None,
+    engine_instances=None,
+    env=None,
+    secrets=None,
+    certificate=None,
+    key_vault_reference_identity=None,
+    subnet_id=None,
+    no_wait=False,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    body = client.get_test(test_id)
+    logger.debug("Retrieved test with test ID: %s and body : %s", test_id, body)
+    body = create_or_update_body(
+        test_id,
+        body,
+        load_test_config_file=load_test_config_file,
+        display_name=display_name,
+        test_description=test_description,
+        engine_instances=engine_instances,
+        env=env,
+        secrets=secrets,
+        certificate=certificate,
+        key_vault_reference_identity=key_vault_reference_identity,
+        subnet_id=subnet_id,
+    )
+    logger.info("Updating test with test ID: %s and body : %s", test_id, body)
+    response_obj = client.create_or_update_test(test_id=test_id, body=body)
+    logger.debug(
+        "Response object for updating test with test ID: %s is %s",
+        test_id,
+        response_obj,
+    )
+    logger.info(
+        "Updated test with test ID: %s and response obj is %s", test_id, response_obj
+    )
+    if test_plan is not None:
+        upload_test_plan(client, test_id, test_plan, no_wait)
+
+    if configuration_files is not None:
+        upload_configuration_files(client, test_id, configuration_files)
+    return response_obj
+
+
+def list_tests(
+    cmd,
+    load_test_resource,
+    resource_group_name=None,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    logger.info("Listing tests...")
+    return client.list_tests()
+
+
+def get_test(
+    cmd,
+    load_test_resource,
+    test_id,
+    resource_group_name=None,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    logger.debug("Retrieving test with test ID: %s", test_id)
+    return client.get_test(test_id)
+
+
+def download_test_files(
+    cmd,
+    load_test_resource,
+    test_id,
+    path,
+    resource_group_name=None,
+):
+    client = get_admin_data_plane_client(cmd, load_test_resource, resource_group_name)
+    list_of_file_details = client.list_test_files(test_id)
+    if list_of_file_details:
+        path = os.path.normpath(os.path.expanduser(path))
+        if not os.path.exists(path):
+            logger.debug("Directory does not exist, creating directory %s", path)
```

Comment:
We do support creating the directory but the validator in path fails if the directory doesn't exist :/ Remove the validation there?

Reply:
Good catch! We want to remove creating the directories for now as it can have unintended effects on users